operator/duties/controller.go

```diff
@@ -241,6 +244,55 @@ func (dc *dutyController) handleSlot(logger *zap.Logger, slot phase0.Slot) {
 		go dc.onDuty(logger, &duties[i])
 	}
 
+	dc.handleSyncCommittee(logger, slot, syncPeriod)
```

Comment:
this isn't related to MEV, you just took out the sync committee code to a different func right?